winrt/lib/text/CanvasFontFace.abi.idl

```diff
@@ -82,6 +82,19 @@ namespace Microsoft.Graphics.Canvas.Text
         float AscenderOffset;
     } CanvasGlyph;
 
+    [version(VERSION)]
+    typedef struct CanvasGlyphMetrics
+    {
+        float LeftSideBearing;
+        float AdvanceWidth;
+        float RightSideBearing;
+        float TopSideBearing;
+        float AdvanceHeight;
+        float BottomSideBearing;
+        float VerticalOriginY;
+        Windows.Foundation.Rect BlackBox;
```

Comment:
Could you please rename this to DrawBounds? That way, it's consistent with our CanvasTextLayout.DrawBounds.